examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1223,7 +1237,7 @@ class GLTFWriter {
 
 				if ( format !== RGBAFormat ) {
 
-					console.error( 'GLTFExporter: Only RGBAFormat is supported.' );
+					console.error( 'GLTFExporter: Only RGBAFormat is supported.', image );
```

Comment:
nit: Usually we log string errors and not objects — Unless you're keeping this for temporary debugging while working on the PR, I think we could either (a) remove the image argument, or (b) log the format instead.

```suggestion
					console.error( 'GLTFExporter: Only RGBAFormat is supported.' );
```